File: org.diverse.PCM/org.diverse.PCM.io.Wikipedia/src/main/scala/org/diverse/pcm/io/wikipedia/pcm/Cell.scala
package org.diverse.pcm.io.wikipedia.pcm

import org.diverse.pcm.api.java.impl.PCMFactoryImpl


class Cell(
  val content : String,
  val isHeader : Boolean,
  val row : Int,
  val rowspan : Int,
  val column : Int,
  val colspan : Int
  ) {
  
  override def toString() : String = {
     content
  }

  def toPCM() : org.diverse.pcm.api.java.Cell = {
    val factory = new PCMFactoryImpl()
    val cell = factory.createCell()
    cell.setContent(content)
    cell
  }
  
}

File: org.opencompare/api-java/src/main/java/org/diverse/pcm/api/java/Cell.java
package org.diverse.pcm.api.java;

public interface Cell extends PCMElement {

	String getContent();
	void setContent(String content);
	
	Value getInterpretation();
	void setInterpretation(Value interpretation);

    Feature getFeature();
    void setFeature(Feature feature);
	
}
